### DISCUSSION TITLE
Dark theme [Jan 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4937] [Discord Threads]
> 🙋‍♂️ **kannthus**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### It was present in earlier release. Was shelved during UI upgrade and hence - currently is not available.  
#  
> 🙋‍♂️ **kannthus**  
#### Oh that’s unfortunate, I’ll await eagerly in hopes it comes to fruition. Is there a PR people could contribute towards on it?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1054551998785146931)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by Torpedro1978** [comment]
A dark theme is mandatory for any application, please add at least a dark theme if full themee support is to much work.

**Reply by ballinger** [reply]
+1 Please add a dark mode or some admin css override. Thanx!
